packages/web-components/src/components/masthead/masthead.scss

```
@@ -152,18 +152,24 @@
 
   &[usage-mode='header-nav'],
   &[collapse-mode][usage-mode='header-nav'] {
+    visibility: hidden;
     width: 0;
+    transition: visibility 0.11s step-end 0s, width 0.11s cubic-bezier(0.2, 0, 1, 0.9) 0s;
   }
 
   &[expanded][usage-mode='header-nav'],
   &[collapse-mode][expanded][usage-mode='header-nav'] {
     @include carbon--breakpoint-down('lg') {
+      visibility: visible;
       width: mini-units(32);
+      transition: visibility 0.11s step-start 0s, width 0.11s cubic-bezier(0.2, 0, 1, 0.9) 0s;
     }
 
     @include carbon--breakpoint-down('md') {
+      visibility: visible;
```

```suggestion
      visibility: inherit;
```

Ref: https://jakearchibald.com/2014/visible-undoes-hidden/